tests/filesystem_tests/test_hdfs_handler.py

```diff
@@ -148,6 +156,9 @@ def test_get_principal_name_from_klist(self):
 
     def test_list(self):
         with chainerio.create_handler(self.fs) as handler:
+            with handler.open(self.tmpfile_name, "w") as tmpfile:
```

Comment:
Can you please move this part in `setUp` to reduce the code redundancy?

Reply:
OK. I will divide the test because some input files are different.